README.md

```diff
@@ -1,13 +1,17 @@
-# Setup and mobile version skeleton
+# Portffolio 
 
->Portfolio setup and mobile version skeleton front-end page.
+>Portfolio setup desktop version and mobile version skeleton front-end page.
 
 
 ## Built With
 
 - HTML
 - CSS
 
+## Live demo
+
+https://ghilain.github.io/mobile-version-skeleton/
+
```

Comment:
- [ ] [OPTIONAL] You can make it like this, maybe it's nicer. But it's up to you.

`[Link](https://nmeziany.github.io/SkeletonMobilePortfolio/)`

Reply:
Thank you for support